shell/platform/android/android_context_gl_impeller_unittests.cc

```diff
@@ -104,6 +104,15 @@ TEST_F(AndroidContextGLImpellerTest, FallbackForEmulator) {
   EXPECT_CALL(
       *display,
       ChooseConfig(Matcher<ConfigDescriptor>(AllOf(
+          Field(&ConfigDescriptor::api, impeller::egl::API::kOpenGLES3),
```

Comment:
@jonahwilliams looks like we could have used these tests for asserting we are asking for a depth buffer.  It had been a while since I looked at these.

Reply:
🤷‍♂️